internal/install/command.go

```diff
@@ -50,7 +50,6 @@ var Command = &cobra.Command{
 
 		detailErr := validateProfile(config.DefaultMaxTimeoutSeconds, sg)
 		if detailErr != nil {
-			sg.Track(detailErr.EventName)
```

Comment:
Assuming we must be tracking this event on [line 124](https://github.com/newrelic/newrelic-cli/blob/fb3caec2b346fe4d7e74217af4ae23b282fe2ec8/internal/install/command.go#L124) then?

Reply:
Exactly. I've removed that one because L124 has more context passed-in.